my-webxr-app/src/App.tsx

```diff
@@ -108,79 +111,81 @@ export default function App() {
         </button>
       </div>
       <VRButton />
-      <PointSelectionProvider>
-        <Canvas>
-          <XR>
-            <Sky sunPosition={[0, 1, 0]} />
-            <Floor />
-            <ambientLight />
-            <pointLight position={[10, 10, 10]} />
-            <Controllers />
+      <Provider config={rollbarConfig}>
```

Comment:
What does the Provider do? Just allow access to the Rollbar logging?

Reply:
Yes, the Provider acts like the PointSelectionProvider and allows components to reference the Rollbar configuration.